guides/_posts/2014-01-5-vfl.html

```diff
@@ -219,31 +233,15 @@
 {% endhighlight %}
 
 
-<h2>Implicit Connections</h2>
-
-Align group elements matching a selector:
-
-{% highlight css %}
-
-@v .section;
-
-@h .menu-item 
-  in(#menu)
-  gap(16)
-  outer-gap(32);
-
-{% endhighlight %}
-
-**WARNING:** only relatively simple selectors without spaces currently work.
-
-
-<h2>Cushions</h2>
+<a name="cushions"></a>
+## Cushions
 
-Cushions are similar to connections, only they use inequality instead of equality constraints. 
+Cushions are similar to connections gaps, only they use inequality instead of equality constraints.
+<a href="http://codepen.io/cbchouinard/pen/JoyQBq" target="_blank">Live example.</a>
 
 {% highlight css %}
 /* VFL */
-@horizontal [#box1]~[#box2];
+@h (#box1)~(#box2);
```

Comment:
I'm not sure what this does. What is `~` when used as a gap? Is this the same as `~-~`?